.github/workflows/azure_kubernetes_aks_single_region_golden.yml

```diff
@@ -89,6 +89,10 @@ jobs:
                   tenant-id: ${{ steps.secrets.outputs.AZURE_TENANT_ID }}
                   subscription-id: ${{ steps.secrets.outputs.AZURE_SUBSCRIPTION_ID }}
 
+            - name: Copy config_override.tf to module dir
+              run: |
+                  cp -r ./azure/kubernetes/aks-single-region/test/fixtures/fixture_config_override.tf ${{ env.MODULE_DIR }}
+
```

Comment:
as part of the script execution the following is done

```
  cd {{ justfile_directory() }}/{{ module_dir }}

  # Copy *.tf files from tests/fixtures/ to the current directory before running the plan
  if ls test/fixtures/fixture_*.tf 1> /dev/null 2>&1; then
    cp test/fixtures/fixture_*.tf ./
  fi
```

do we then need to explicitly copy the file here?
It will create eventually the case where one may wonder why we copy this file but not the other files etc.